tests/acceptance/TestHelpers/AuthAppHelper.php

```diff
@@ -62,16 +62,19 @@ public static function listAllAppAuthTokensForUser(
 	 * @param string $user
 	 * @param string $password
 	 * @param string $expiration
+	 * @param string $impersonatorUser
 	 *
 	 * @return ResponseInterface
 	 */
 	public static function createAppAuthToken(
 		string $baseUrl,
 		string $user,
 		string $password,
-		string $expiration
+		string $expiration,
+		string $impersonatorUser = ''
```

Comment:
send params as an array and build it using: https://www.php.net/manual/en/function.http-build-query.php
```suggestion
		array $params
```